react/README.md

```diff
@@ -18,3 +29,259 @@ For more info, see [Getting started with React](https://facebook.github.io/react
 [JSX](https://facebook.github.io/jsx/) is a JavaScript syntax extension that looks similar to XML. We use it with React because it is a concise and familiar syntax for defining tree structures with attributes.
 
 For more info, see [JSX in Depth](https://facebook.github.io/react/docs/jsx-in-depth.html).
+
+## General rules
+
+- Always use JSX syntax (don't use `React.createElement`)
```

Comment:
Maybe clarify a bit here what JSX syntax would look like / what exactly is the bad example an example of? I understand what you are saying but might be good for clarity.